testing/src/main/java/io/quarkiverse/githubapp/testing/internal/CallRealMethodAndSpyGHObjectResults.java

```diff
@@ -20,6 +20,7 @@ public CallRealMethodAndSpyGHObjectResults(GitHubMockContextImpl mocks) {
     @Override
     public Object answer(InvocationOnMock invocation) throws Throwable {
         Object original = invocation.callRealMethod();
+        String packageName = original.getClass().getPackage().getName();
```

Comment:
This looks like a useless change?

Reply:
It definitely is. Some remaining debug code. Will delete it.